crates/rspack_plugin_javascript/src/visitors/dependency/common_js_export_scanner.rs

```diff
@@ -182,6 +185,32 @@ impl Visit for CommonJsExportDependencyScanner<'_> {
             // this = require('xx');
             // It's possible to reexport __esModule, so we must convert to a dynamic module
             self.set_dynamic();
+            let related_require_dep = self
```

Comment:
A little hack, webpack will skip rest visiting if found a `CommonjsExportRequireDependency`, but we always generate that dependency, need parser hook.  https://github.com/webpack/webpack/blob/1f99ad6367f2b8a6ef17cce0e058f7a67fb7db18/lib/dependencies/CommonJsExportsParserPlugin.js#L207-L219